delta/plugins/blazegraph/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/blazegraph/model/BlazegraphViewEvent.scala

```diff
@@ -43,6 +44,11 @@ sealed trait BlazegraphViewEvent extends Event {
     * @return the revision that the event generates
     */
   def rev: Long
+
+  /**
+    * @return the type of this event
+    */
+  def eventType: String = BlazegraphViews.moduleType
```

Comment:
Maybe we can have a custom type for the moduleType (eventType) concept?

Something like 

```scala 
trait ModuleType {
   def name: String
}

object ModuleType {
   final case object ProjectModule extends ModuleType("project")
   // others
}
```

Reply:
I don't know if using the module type is the best idea in the long run as these are two different things.
Sure for now, it fits as we have a one-to-one relationship but this may change.